src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clearObject = [];
```

Comment:
It's not object it's array. Improve variable naming, `result` for example
```suggestion
  const clearObject = [];
```